src/streamlink/plugins/afreeca.py

```diff
@@ -207,10 +213,7 @@ def _get_streams(self):
 
         channel = self._get_channel_info(bno, username)
         log.trace(f"{channel!r}")
-        if channel.get("BPWD") == "Y":
-            log.error("Stream is Password-Protected")
-            return
-        elif channel.get("RESULT") == -6:
+        if channel.get("RESULT") == -6:
```

Comment:
Please add a constant to the class, so that the `-6` doesn't randomly appear in the middle of `_get_streams()`.

Reply:
I was confused and it was fixed in #5953